pyanaconda/payload/dnf/payload.py

```diff
@@ -563,6 +557,23 @@ def _configure_proxy(self):
             config.proxy_username = None
             config.proxy_password = None
 
+    def _get_proxy_url(self):
```

Comment:
Please rename this to `_get_url_source_proxy`.

Reply:
It is not a DBus proxy, but the URL proxy.